lib/KalturaHelpers.php

```diff
@@ -200,8 +200,8 @@ public static function getOption( $name, $default = null ) {
 		$name    = is_string( $name ) ? $name : null;
 		$default = is_bool( $default ) ? $default : null;
 
-		$value = get_option( $name, $default );
-		if ( ! is_null( $value ) ) {
+		$value = self::isPluginNetworkActivated() ? get_site_option( $name, $default ) : get_option( $name, $default );
```

Comment:
But if there is already a config in a specific instance?
For instance, if my site started as a single site and then I made it multisite - then the original site will have an option and it won't be overwritten by the 'meta' site options...